<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShoushiUIClass</class>
 <widget class="QWidget" name="ShoushiUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="shoushiWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>220</width>
     <height>230</height>
    </rect>
   </property>
   <widget class="QPushButton" name="shouYuanhuaBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>15</y>
      <width>47</width>
      <height>47</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/beauty/beauty_original);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="shou1Btn">
    <property name="geometry">
     <rect>
      <x>57</x>
      <y>15</y>
      <width>47</width>
      <height>47</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/beauty/gesture_666);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="shou2Btn">
    <property name="geometry">
     <rect>
      <x>114</x>
      <y>15</y>
      <width>47</width>
      <height>47</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/beauty/gesture_baoquan);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="shou3Btn">
    <property name="geometry">
     <rect>
      <x>171</x>
      <y>15</y>
      <width>47</width>
      <height>47</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/beauty/gesture_heshi);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="shou7Btn">
    <property name="geometry">
     <rect>
      <x>171</x>
      <y>80</y>
      <width>47</width>
      <height>47</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/beauty/gesture_thumbsup);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="shou5Btn">
    <property name="geometry">
     <rect>
      <x>57</x>
      <y>80</y>
      <width>47</width>
      <height>47</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/beauty/gesture_onehandheart);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="shou4Btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>47</width>
      <height>47</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/beauty/gesture_ok);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="shou6Btn">
    <property name="geometry">
     <rect>
      <x>114</x>
      <y>80</y>
      <width>47</width>
      <height>47</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/beauty/gesture_palm);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="shou8Btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>145</y>
      <width>47</width>
      <height>47</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/beauty/gesture_tuojv);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="shou11Btn">
    <property name="geometry">
     <rect>
      <x>171</x>
      <y>145</y>
      <width>47</width>
      <height>47</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/beauty/gesture_zhizhu);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="shou9Btn">
    <property name="geometry">
     <rect>
      <x>57</x>
      <y>145</y>
      <width>47</width>
      <height>47</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/beauty/gesture_twohandheart);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="shou10Btn">
    <property name="geometry">
     <rect>
      <x>114</x>
      <y>145</y>
      <width>47</width>
      <height>47</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/beauty/gesture_yeah);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../joy.qrc"/>
 </resources>
 <connections/>
</ui>
